modules/mod_admin/lib/css/zotonic-admin.css

```diff
@@ -111,9 +115,6 @@ body.noframe > .admin-container {
 .admin-edit-media {
   border: 1px solid #fff;
 }
-iframe, video, audio {
-  max-width: 100%;
-}
```

Comment:
That code is not part of a LESS file, so on recompile that code is removed from the CSS.